### DISCUSSION TITLE
Scroll using keyboard keys

### DISCUSSION THREAD

**Discussion by rahullalw** [pieces-app/support#98] [Ideas]
Enable navigation through the page using the upward and downward arrow keys as well as the Page Up and Page Down keys.

**Comment by caleb-at-pieces** [comment]
Could you elaborate a bit more on the specifics here?

The feature is supposed to allow you to changes views within the desktop app with these specified keys? I think this wouldn't be all that complicated, our desktop app developer @mack-at-pieces can comment here.
